h2o-algos/src/test/java/hex/glm/GLMBasicTestMultinomial.java

```diff
@@ -50,6 +52,36 @@ public static void cleanUp() {
     if(_train != null) _train.delete();
     if(_test != null) _test.delete();
   }
+  
+  @Test
+  public void testcalculateVarimpMultinomial() {
+    int numClass = 5;
+    int coeffLen = 25;
+    double[][] betaNorm = new double[numClass][coeffLen];
+    double[] magnitudes = new double[coeffLen];
+    double[] magnitudes_check = new double[coeffLen];
+    int[] indices = new int[coeffLen];
+    Random randObj = new Random(12345);
+    
+    for (int indexc = 0; indexc < numClass; indexc++) { // generate random double[][] array
+      for (int index = 0; index < coeffLen; index++) {
+        betaNorm[indexc][index] = randObj.nextDouble();
+      }
+    }
+    calculateVarimpMultinomial(magnitudes, indices, betaNorm);  // calculate magnitude and generate indices of descending sort
+    
+    for (int classInd = 0; classInd < numClass; classInd++) { // manually generate coefficient magnitudes
+      for (int coeffInd = 0; coeffInd < coeffLen; coeffInd++) {
+        magnitudes_check[coeffInd] = magnitudes_check[coeffInd] + Math.abs(betaNorm[classInd][coeffInd]);
+      }
+    }
+    // check to make sure magnitude calculation is correct
+    assert Arrays.equals(magnitudes, magnitudes_check) : "Magnitude calculation for multinomial is incorrect.";
```

Comment:
Please junit assert instead of Java assert (they can be disabled!).

Reply:
Sure

Reply:
Good point.  Done!